### DISCUSSION TITLE
01. Entretenir son site régulièrement

### DISCUSSION THREAD

**Discussion by dNicolle** [cnumr/best-practices-wordpress#47] [Bonnes pratiques] [answered]
https://github.com/cnumr/fiches-wordpress-greenit/blob/main/best-practices-WordPress/fiches/01.%20Entretenir%20son%20site%20r%C3%A9guli%C3%A8rement.md

@florinesueur propose de spliter cette BP

**Comment by florinesueur** [comment]
Hello,
Êtes vous d'accord pour découper cette BP ?

 Si oui, je vous propose :
- Supprimer les pages ou contenus non utilisés ( supprimer régulièrement les pages inutiles ou tout contenu obsolète + supprimer les fichiers multimédias inutilisés stockés en bibliothèque)
- Supprimer les thèmes et plugins non utilisés
- Procéder à la mise à jour régulière du thème et des extensions
- Nettoyer base de données et anciennes sauvegardes

**Comment by florinesueur** [comment, marked as answer]
La BP est écrite.
Issue : https://github.com/cnumr/fiches-wordpress-greenit/issues/137
